.travis.yml

```diff
@@ -1,21 +1,22 @@
 language: go
 
 go:
-  - '1.11'
+  - '1.12.x'
```

Comment:
As this is a library, should we preserve compatibility with 1.11? Obviously we can do what we want in the CI, but it occurs to me we might want both versions in the test matrix.

Reply:
This is an interesting case because it's a C library, not Go library. It may be considered an application since we only provide binary releases.

Reply:
Makes sense. I guess if the end user is typically going to see a binary release anyway, they won't be held up by the Go version they're using.